lib/plugins.js

```diff
@@ -771,6 +771,10 @@ exports.categories = {
             url: 'https://github.com/roylines/hapi-graceful-pm2',
             description: 'Handle true zero downtime reloads when issuing a pm2 gracefulReload command'
         },
+        'hapi-openapi': {
+            url: 'https://github.com/krakenjs/hapi-openapi',
+            description: 'Hapi plugin to build design-driven apis with OpenAPI (formerly swagger).'
```

Comment:
Only nit is that "hapi" is intended to always be styled all-lowercase.  I know there are a small handful of descriptions in here that break that, but figured it wouldn't hurt to mention!